lib/metadataTypes/Asset.js

```diff
@@ -162,22 +162,21 @@ class Asset extends MetadataType {
         } else {
             Util.logger.info(`- Caching Subtype: ${subType}`);
         }
-        const subtypeIds = subTypeArray?.map(subTypeItemName => Asset.definition.typeMapping[subTypeItemName]);
-        const uri = 'asset/v1/content/assets/';
-        const options = {
-            uri: uri + 'query',
-            headers: {},
-            json: {
-                page: {
-                    page: 1,
-                    pageSize: 50,
-                },
-                query: null,
-                fields: [],
+        const subtypeIds = subTypeArray?.map(
+            (subTypeItemName) => Asset.definition.typeMapping[subTypeItemName]
+        );
+        const uri = 'asset/v1/content/assets/query';
```

Comment:
That is because the other changes were simple cases. in this case we have a main request, supported by sub requests.  Assets basically requires some more custom handling.

Reply:
lets have a quick chat about this one. I want to understand this change better :)

- [ ] have a call for this

Reply:
push @DougMidgley